banners/DE/pi30.json

```
@@ -5,21 +5,17 @@
         "offerTerm": "{total_payments}",
         "lander": "",
         "variables": {
-            "offer_id": "${CREDIT_OFFERS_DS.offer_id}",
-            "vendor_financing_id": "${CREDIT_OFFERS_DS.vendor_financing_id}",
             "financing_code": "${CREDIT_OFFERS_DS.financing_code}",
+            "formattedMonthlyPayment": "${CREDIT_OFFERS_DS.formattedMonthlyPayment}",
             "total_payments": "${CREDIT_OFFERS_DS.total_payments}",
             "formattedMinAmount": "${CREDIT_OFFERS_DS.formattedMinAmount}",
-            "formattedMaxAmount": "${CREDIT_OFFERS_DS.formattedMaxAmount}"
+            "formattedTotalCost": "${CREDIT_OFFERS_DS.formattedTotalCost}"
         }
     },
-    "headline": [
-        ["Jetzt kaufen, später bezahlen.", ["default", "xsmall"]],
-        [
-            ["Bezahlen Sie bei Einkäufen von {formattedMinAmount} bis {formattedMaxAmount} erst nach 30 Tagen"],
-            ["medium", "large", "xlarge"]
-        ]
-    ],
+
+    "headline": [["Jetzt kaufen, später bezahlen.", ["default", "small"]]],
```

```suggestion
    "headline": [["Jetzt kaufen, später bezahlen.", ["default", "xsmall"]]],
```
Let's change this to be xsmall so it matches the other messages.